src/main/app/src/components/Login.js

```diff
@@ -0,0 +1,71 @@
+import { Button, Grid, TextField, makeStyles, Paper, useMediaQuery, useTheme } from '@material-ui/core';
```

Comment:
This file is formatted with 4 space indents (I know it was Justin who did this file), can you change the indentation to 2 spaces? So that it matches the rest of the javascript files